pkg/platform/abstract/platform.go

```diff
@@ -447,12 +447,39 @@ func (ap *Platform) GetProcessorLogsAndBriefError(scanner *bufio.Scanner) (strin
 		formattedProcessorLogs += currentLogLine + "\n"
 	}
 
+	*briefErrorsArray = ap.aggregateConsecutiveDuplicateMessages(*briefErrorsArray)
+
 	// create brief errors log as string, and remove double newlines
 	briefErrorsMessage = strings.Join(*briefErrorsArray, "\n")
 
 	return common.FixEscapeChars(formattedProcessorLogs), common.FixEscapeChars(briefErrorsMessage)
 }
 
+func (ap *Platform) aggregateConsecutiveDuplicateMessages(errorMessagesArray []string) []string {
+	var aggregatedErrorsArray []string
+
+	for i := 0; i < len(errorMessagesArray); i++ {
```

Comment:
You can use range loop on `errorMessagesArray` here, that will saves you `currentErrorMessage := errorMessagesArray[i]`

Reply:
The internal index modification won't work as expected with a range loop